src/sst/elements/memHierarchy/mshr.cc

```diff
@@ -412,19 +414,19 @@ bool MSHR::insertEviction(Addr oldAddr, Addr newAddr) {
         reg.entries_.push_back(MSHREntry(newAddr, getCurrentSimCycle()));
         mshr_.insert(std::make_pair(oldAddr, reg));
     } else {
-        list<MSHREntry>* entries = &(mshr_.find(oldAddr)->second.entries_);
-        if (!entries->empty() && entries->back().getType() == MSHREntryType::Evict) { // MSHR entry for oldAddr is an Evict
-            entries->back().getPointers()->push_back(newAddr);
+        list<MSHREntry>* entries_ = &(mshr_.find(oldAddr)->second.entries_);
```

Comment:
Revert "entries_" to "entries" in this function since it's a local var not the class member